workflow-parser/src/model/workflow-template.ts

```diff
@@ -1,3 +1,4 @@
+import { Scalar } from "yaml";
```

Comment:
This import isn't used, which will show up as a warning in the linter and not a value

`npm run lint`

```
  1:10  warning  'Scalar' is defined but never used  @typescript-eslint/no-unused-vars
```